validator_test.go

```diff
@@ -19,8 +19,8 @@ package validator_test
 import (
 	"testing"
 
+	"github.com/wcl48/go-validator"
```

Comment:
I'm assuming you changed your imports for testing and forgot to change it back. Please do it and I'll merge.